documentation/API_documentation/NumberVerification_device_phone_number_User_Story.md

```diff
@@ -0,0 +1,10 @@
+# Number Verification Device Phone Number API User Story
```

Comment:
```suggestion
# Number Verification Retrieve Device Phone Number API User Story
```

Reply:
Ok with the change, but "Share" would be more accurate as it's the terms used in the yaml

Reply:
I'm fine with share - the yaml uses also return.
We need to close this one quickly with the closure of the release ;)